core/src/main/java/com/graphhopper/routing/weighting/custom/EnumToValueEntry.java

```diff
@@ -38,6 +41,66 @@ public double getValue(EdgeIteratorState iter, boolean reverse) {
         return values[enumOrdinal];
     }
 
+    /**
+     * Example map:
+     * <pre>
+     * road_class:
+     *   motorway: 0.4
+     *   "*": 0.9      # optional and default is 1
+     * </pre>
+     */
+    static EnumToValueEntry create(String name, EnumEncodedValue enumEncodedValue, Map<Object, Object> map,
```

Comment:
Thats very nice we have these create methods now rather than doing it in speed/priorityCalculator's constructor